src/sagas/pushNotification.js

```diff
@@ -434,6 +456,58 @@ export function* dismissOptInQuestion() {
   yield STORE.setItem(pushNotificationKey.optInDismissed, true);
 }
 
+/**
+ * This function is responsible for checking if the app was opened by a push notification
+ * and if so, it will load the tx and token data to show the tx detail modal.
+ */
+export function* checkOpenPushNotification() {
+  try {
+    /**
+     * @example
+     * {
+     *  notification: {
+     *    id: '36f30UJBYhxiIOgVzAft',
+     *    title: 'New transaction received',
+     *    body: 'You have received 10 T2, 5 T1 and 2 other token on a new transaction.',
+     *    data: {
+     *      tx: <stringfied>{
+     *        txId: '000021e7addbb94a8e43d7f1237d556d47efc4d34800c5923ed3a75bf5a2886e',
+     *        timestamp: 1673039453,
+     *        balance: 500,
+     *        voided: false,
+     *        tokenUid: '00',
+     *      }
+     *      token: <stringfied>{
+     *        name: 'Hathor',
+     *        symbol: 'HTR',
+     *        uid: '00',
+     *      }
+     *    },
```

Comment:
@pedroferreira1, here is what I want to discuss. Why not pass the main token in the notification's metadata?